exercises/Cielo-Raymundo/answers.js

```diff
@@ -0,0 +1,123 @@
+// Question 1
+function greet(name) {
+  let answer=name;
+  return(`Good morning, ${answer}!`);
+}
+
+// Question 2
+function logOddNumbers() {
+  for(let i=1; i<100;i++){
+    if(i%2!=0){
+      console.log(i);
+    }
+  }
+}
+
+// Question 3
+function area(length, width) {
+  return length*width;
+}
+
+// Question 4
+function otherAngle(angle, angle2) {
+  let angle3;
+  if(angle>0&&angle2>0){
+    angle3= 180-(angle + angle2);
+  }
+  return angle3;
+}
+
+// Question 5
+function diameterFromArea(area) {
+  let term1=area/Math.PI;
+  let diameter = 2*(Math.sqrt(term1));
+  return diameter;
+}
+
+// Question 6
+function totalPrice(price, tax) {
+  let totalTax= (price * tax)/100;
+  let total= totalTax+price;
+  return total;
+}
+
+// Question 7
+function shortLongShort(string1, string2) {
```

Comment:
we don't need to declare the variables `long` and `short`. We can just use `string1` and `string2`. As parameters, the become locally scoped variables in the function body.